server.go

```diff
@@ -56,25 +56,22 @@ type Server struct {
 	disableAutoGroupTags   bool
 	basePath               string // Base path of the group
 
-	// Points to the server OpenAPI struct.
-	OpenAPI *OpenAPI
+	*Engine
 
 	Security Security
 
 	autoAuth AutoAuthConfig
 	fs       fs.FS
 	template *template.Template // TODO: use preparsed templates
 
-	acceptedContentTypes []string
-
 	DisallowUnknownFields bool // If true, the server will return an error if the request body contains unknown fields. Useful for quick debugging in development.
 	DisableOpenapi        bool // If true, the routes within the server will not generate an OpenAPI spec.
 	maxBodySize           int64
 
-	Serialize      Sender                // Custom serializer that overrides the default one.
-	SerializeError ErrorSender           // Used to serialize the error response. Defaults to [SendError].
-	ErrorHandler   func(err error) error // Used to transform any error into a unified error type structure with status code. Defaults to [ErrorHandler]
-	startTime      time.Time
+	Serialize      Sender      // Custom serializer that overrides the default one.
+	SerializeError ErrorSender // Used to serialize the error response. Defaults to [SendError].
```

Comment:
I guess while we're here. Would you mind sending all the comments above the fields? If not approved

Reply:
Done! I did not do it immediately for readability but I thought about it ;)